### PR TITLE
Getting-started tweaks

### PR DESCRIPTION
**Pull request by raboof**
I think I covered most of the 'loose ends' from comments
on #22621. 'quickstart' still needs to be updated to
use sbt-new/g8.

### CODE REVIEW

File: akka-docs-new/src/main/paradox/guide/introduction.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ efficiently.
 ## What is the actor model?
 
 The characteristics of today's computing environments are vastly different from the ones in use when the programming 
-models of yesterday were conceived. Actors were invented decades ago by Carl Hewitt (reference).
+models of yesterday were conceived. Actors were invented decades ago by @extref[Carl Hewitt](wikipedia:Carl_Hewitt#Actor_model).
```
Comment (patriknw) [inline akka-docs-new/src/main/paradox/guide/introduction.md]: nice

File: akka-docs-new/src/main/paradox/guide/modules.md [hunk 2]
```diff
@@ -111,6 +111,20 @@ Persistence tackles the following problems:
...
+In situations where eventual consistency is acceptable it is possible to share data between nodes in
+an Akka Cluster and accept both reads and writes even in the face of cluster partitions. This can be
+achieved using Conflict Free Replicated Data Types (CRDTs), where writes on different nodes can
+happen concurrently and are merged in a predictable way afterwards. The Distributed Data module
+provides infrastructure to share data and a number of useful data types.
+
+Distributed Data is intended to solve the following challenges:
+
+* How can I accept writes even in the face of cluster partitions?
+* How can I share data while at the same time ensuring low-latency local read and write access?
```
Comment (patriknw) [inline akka-docs-new/src/main/paradox/guide/modules.md]: good

**Review by patriknw — APPROVED** [review]
LGTM, squash before merging (since all those commits are rather related)

**Comment by raboof** [comment]
@patriknw githubs 'squash and merge' web ui button should do the right thing here, right? Or is there a reason to do it 'manually'?

**Comment by patriknw** [comment]
That works fine here. There are situations when it's not doing the right thing (or at least didn't previously), such as when multiple authors contribute to the same PR.

File: akka-docs-new/src/main/paradox/guide/modules.md [hunk 3]
```diff
@@ -119,7 +133,7 @@ to implement the same pattern over and over. Very common is the scenario where a
 process a potentially large (or infinite) stream of sequential events and properly coordinate resource usage so that 
 faster processing stages don’t overwhelm slower ones in the chain (or graph). Streams provide a higher-level 
 abstraction on top of actors that simplifies writing such processing networks, handling all the fine details in the 
-background and providing a safe programming model. Streams is also an implementation 
+background and providing a safe (typed), composable programming model. Streams is also an implementation
```
Comment (johanandren) [inline akka-docs-new/src/main/paradox/guide/modules.md]: Why the parenthesises? "safe, typed, composable". Safe in the sense that it protects against doing the wrong things in runtime as well as through the type system (`GraphStage` provides guard rails)

**Review by johanandren — APPROVED** [review]
LGTM with one comment

**Comment by 2m** [comment]
LGTM

File: akka-docs-new/src/main/paradox/guide/modules.md (hunk 3 above)
Comment (raboof) [inline akka-docs-new/src/main/paradox/guide/modules.md]: I like it!

**Comment by johanandren** [comment]
Yay, first merge! 🥇

**Pull request merged**